### PR TITLE
Revised data content structure for each kinesis stream, making it eas…

### PR DESCRIPTION
**Pull request by wozzer72**
…y to consume.

https://trello.com/c/WzrKGR3M

### CODE REVIEW

**Comment by wozzer72** [comment]
[Kinesis Streams - establishments, workers and users data presented in wrong structure, making it hard to consume.](https://trello.com/c/WzrKGR3M/233-kinesis-streams-establishments-workers-and-users-data-presented-in-wrong-structure-making-it-hard-to-consume)

File: server/aws/kinesis.js [hunk 1]
```diff
@@ -56,7 +59,7 @@ const workerPump = async (action, worker)  => {
   delete worker.postcode;
 
   const params = {
-    Data: JSON.stringify(worker),
+    Data: JSON.stringify(pumpData),
     PartitionKey: worker.uid,    // establishment's primary key
```
Comment (nasirmids) [inline server/aws/kinesis.js]: Is it not pumpData.worker.uid?

Comment (nasirmids) [inline server/aws/kinesis.js]: Minor: comment changes from establishment primary key to worker primary key

Comment (wozzer72) [reply server/aws/kinesis.js]: It can be. But `worker` is a passed in parameter, and hence `worker.uid` is still valid.

Comment (wozzer72) [reply server/aws/kinesis.js]: Too true; this has been in there for a while.

Comment (wozzer72) [reply server/aws/kinesis.js]: I've corrected the comment here for Worker and the one for User below too.

Comment (nasirmids) [reply server/aws/kinesis.js]: ok

Comment (nasirmids) [reply server/aws/kinesis.js]: ok thanks

**Pull request merged**